CoreScriptsRoot/Modules/VR/Panel3D.lua

```diff
@@ -1029,6 +1044,9 @@ local function onWorkspaceChanged(prop)
 			cameraChangedConnection:disconnect()
 		end
 		cameraChangedConnection = workspace.CurrentCamera.Changed:connect(onCameraChanged)
+
+		--Make sure the new camera gets used properly
+		onVREnabled("VREnabled")
```

Comment:
This is a bit misleading of a function invocation. We aren't actually enabling VR here or changing the state of VREnabled at all, rather we are trying refresh what is happening in that function. I would rather the onVREnabled function listen to an event that the CurrentCamera has changed, and stop listening when VREnabled is set to false.